python-frontend/src/main/java/org/sonar/plugins/python/api/tree/Tree.java

```diff
@@ -246,7 +246,10 @@ enum Kind {
     KEY_VALUE_PATTERN(KeyValuePattern.class),
     TOKEN(Token.class),
     LINE_MAGIC(LineMagic.class),
-    LINE_MAGIC_STATEMENT(LineMagicStatement.class);
+    LINE_MAGIC_STATEMENT(LineMagicStatement.class),
+    DYNAMIC_OBJECT_INFO_STATEMENT(DynamicObjectInfoStatement.class),
```

Comment:
Probably you can replace the comma here by semicolon and remove the extra one from line 252.

Reply:
done